src/library/Uno.Toolkit.Material/Styles/Controls/v2/NavigationBar.xaml

```diff
@@ -20,381 +20,873 @@
+						</Grid>
+						<!-- UNO WORKAROUND: Added IsTabStop="False". Navigation focus with keyboard was erroneously focusing on the Popup instead of the SecondaryCommands -->
+						<Popup x:Name="OverflowPopup">
+							<Popup.RenderTransform>
+								<TransformGroup>
+									<TranslateTransform x:Name="OverflowContentRootMarginOffsetTransform" />
+									<TranslateTransform x:Name="OverflowContentRootTransform"
+														X="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=CommandBarTemplateSettings.OverflowContentHorizontalOffset}" />
+								</TransformGroup>
+							</Popup.RenderTransform>
+							<Grid x:Name="OverflowContentRoot"
+								  Opacity="0"
+								  MinWidth="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=CommandBarTemplateSettings.OverflowContentMinWidth}"
+								  MaxWidth="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=CommandBarTemplateSettings.OverflowContentMaxWidth}"
+								  MaxHeight="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=CommandBarTemplateSettings.OverflowContentMaxHeight}">
+								<Grid.RowDefinitions>
+									<RowDefinition />
+									<RowDefinition Height="Auto" />
+								</Grid.RowDefinitions>
+								<!-- For 21H1 and up, we'll need to wrap the clip and presenter in a Grid which will host the Drop Shadow. -->
+								<Grid x:Name="SecondaryItemsControlShadowWrapper"
+									  contract7Present:CornerRadius="{ThemeResource OverlayCornerRadius}"
+									  contract7NotPresent:CornerRadius="{ThemeResource OverlayCornerRadius}">
+									<Grid.Clip>
+										<RectangleGeometry Rect="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=CommandBarTemplateSettings.OverflowContentClipRect}">
+											<RectangleGeometry.Transform>
+												<TransformGroup>
+													<TranslateTransform x:Name="OverflowContentRootClipTransform" />
+												</TransformGroup>
+											</RectangleGeometry.Transform>
+										</RectangleGeometry>
+									</Grid.Clip>
+									<CommandBarOverflowPresenter x:Name="SecondaryItemsControl"
+																 Style="{TemplateBinding CommandBarOverflowPresenterStyle}"
+																 IsTabStop="False">
+										<CommandBarOverflowPresenter.RenderTransform>
+											<TranslateTransform x:Name="OverflowContentTransform" />
+										</CommandBarOverflowPresenter.RenderTransform>
+										<CommandBarOverflowPresenter.ItemContainerStyle>
+											<Style TargetType="FrameworkElement">
+												<Setter Property="HorizontalAlignment" Value="Stretch" />
+												<Setter Property="Width" Value="NaN" />
+											</Style>
+										</CommandBarOverflowPresenter.ItemContainerStyle>
+									</CommandBarOverflowPresenter>
+								</Grid>
+
+								<!--
+									In order to give us extra space in the windowed popup to translate things down,
+									we add a rectangle to make the HWND taller than it otherwise would be.
```

Comment:
to be pedantic, HWND is short for handle of window, not the same as window

Reply:
must've been a copy paste from the actual CommandBar template